### ISSUE TITLE
Arista AVD Version - v4.2.0-dev2 - AVD Runner failed due to eos version mismatch

### ISSUE THREAD

**Issue by duddis1** [ansible-collections/arista.eos#427]
##### SUMMARY
Arista AVD Version - v4.2.0-dev2 - AVD runner failed due to eos version mismatch

##### ISSUE TYPE
Bug

##### COMPONENT NAME
arista.eos

##### COLLECTION VERSION
6.0.1

##### CONFIGURATION
We are getting below error message highlighted while running AVD for the second issue which was reported, due to this we can proceed further.

Do you want us to raise separate ticket in eos space to correct this error in their codebase?
https://github.com/ansible-collections/arista.eos

ERROR LOG:-
```
TASK [arista.avd.eos_designs : Verify Requirements] ************************************************************************************************************************************************************************************
task path: /home/avd/.ansible/collections/ansible_collections/arista/avd/roles/eos_designs/tasks/main.yml:2
Not a git repository
AVD version 4.2.0-dev2
Found netaddr 0.8.0 installed!
Found Jinja2 3.1.2 installed!
Found treelib 1.6.4 installed!
Found cvprac 1.3.1 installed!
Found paramiko 3.2.0 installed!
Found jsonschema 4.17.3 installed!
Found requests 2.28.1 installed!
Found PyYAML 6.0.1 installed!
Found md-toc 8.1.9 installed!
Found deepmerge 1.1.0 installed!
Found cryptography 41.0.2 installed!
### ERROR! arista.eos version running 6.0.0 - required version is >=6.0.1
{
"ansible": {
"collection": {
"name": "arista.avd",
"path": "/home/avd/.ansible/collections/ansible_collections",
"version": "4.2.0-dev2"
},
"ansible_version": "2.14.8",
"requires_ansible": "!=2.13.0,<2.15.0,>=2.12.6",
"collection_requirements": {
"not_found": {},
"valid": {
"arista.cvp": {
"installed": "3.6.1",
"required_version": ">=3.6.1"
},
"ansible.utils": {
"installed": "2.10.3",
"required_version": ">=2.10.3"
}
},
"mismatched": {
"arista.eos": {
"installed": "6.0.0",
"required_version": ">=6.0.1"
}
},
"parsing_failed": []
}
},
"python": {
"python_version_info": {
"major": 3,
"minor": 9,
"micro": 16,
"releaselevel": "final",
"serial": 0
},
"python_path": [
"/home/avd/.local/share/virtualenvs/netbox_extract-zMJz_RXI/bin",
"/usr/lib64/python39.zip",
"/usr/lib64/python3.9",
"/usr/lib64/python3.9/lib-dynload",
"/home/avd/.local/share/virtualenvs/netbox_extract-zMJz_RXI/lib64/python3.9/site-packages",
"/home/avd/.local/share/virtualenvs/netbox_extract-zMJz_RXI/lib/python3.9/site-packages"
],
"python_requirements": {
"not_found": {},
"valid": {
"netaddr": {
"installed": "0.8.0",
"required_version": ">=0.7.19"
},
"Jinja2": {
"installed": "3.1.2",
"required_version": ">=2.11.3"
},
"treelib": {
"installed": "1.6.4",
"required_version": ">=1.5.5"
},
"cvprac": {
"installed": "1.3.1",
"required_version": ">=1.3.1"
},
"paramiko": {
"installed": "3.2.0",
"required_version": ">=2.7.1"
},
"jsonschema": {
"installed": "4.17.3",
"required_version": "<4.18,>=4.5.1"
},
"requests": {
"installed": "2.28.1",
"required_version": ">=2.25.1"
},
"PyYAML": {
"installed": "6.0.1",
"required_version": ">=5.4.1"
},
"md-toc": {
"installed": "8.1.9",
"required_version": ">=7.1.0"
},
"deepmerge": {
"installed": "1.1.0",
"required_version": ">=1.1.0"
},
"cryptography": {
"installed": "41.0.2",
"required_version": ">=38.0.4"
}
},
"mismatched": {},
"parsing_failed": []
}
}
}
fatal: [SVP-CAMP01 -> localhost]: FAILED! => {
"changed": false,
"msg": "Set 'avd_ignore_requirements=True' to ignore validation error(s)."
}
```

Below are the collection list returned by Galaxy.

```
[avd@90d0b1f8adb1 netbox_extract]$ ansible-galaxy collection list

/home/avd/.ansible/collections/ansible_collections
Collection Version

ansible.netcommon 5.1.2
ansible.utils 2.10.3
arista.avd 4.2.0-dev2
arista.cvp 3.6.1
arista.eos 6.0.0
community.hashi_vault 3.4.0
```

##### STEPS TO REPRODUCE

We are unable to generate the intended config with Arista AVD Version "4.2.0-dev2". We have identified the root cause of the issues which we could not resolve, and this will require a change in your code base. 

Can this issue be resolved as a priority because this is blocking the generation of intended config? if you need any further details don't hesitate to contact me, alternatively we can arrange a call as well.

In arista-eos, the version mismatched between the tag and galaxy.yml, as shown in the below screenshot.
https://gitlab.ros.gov.uk/srt-networks/ansible-collections/arista-eos/-/blob/6.0.1/galaxy.yml

##### EXPECTED RESULTS
The arista.eos version should be aligned between the tag and galaxy.yml

##### ACTUAL RESULTS
The arista.eos version currently not aligned between the tag and galaxy.yml

**Comment by ClausHolbechArista** [comment]
To simplify this, the issue re `arista.eos` is that the `galaxy.yml` is showing version 6.0.0 for the released version 6.0.1.

**Comment by duddis1** [comment]
Hi Team, Can we get an ETA on this ticket, currently we are using a workaround to tackle this issue.

Please update us once this issue is resolved, so that we can test our config with the latest changes. Thanks!

**Comment by duddis1** [comment]
@rohitthakur2590  Still i can see version is mentioned as 6.0.0 on galaxy.yml.
https://gitlab.ros.gov.uk/srt-networks/ansible-collections/arista-eos/-/blob/6.0.1/galaxy.yml

Can you please advise when this will be corrected to 6.0.1?

**Comment by duddis1** [comment]
This ticket can be closed, as this is fixed in recent eos version.
